build.sbt

```diff
@@ -101,9 +101,9 @@ lazy val root = (project in file("."))
 
 inScope(Global)(
   Seq(
-    scalaVersion := "2.13.3",
+    scalaVersion := "2.13.4",
     semanticdbEnabled := true,
-    semanticdbVersion := scalafixSemanticdb.revision,
+    semanticdbVersion := "4.4.7",
```

Comment:
`scalafixSemanticdb` this version should be updated automatically with a new scalafix version. Do we really need to change this?

Reply:
without it fails on compile, i will try

Reply:
`[error] sbt.librarymanagement.ResolveException: Error downloading org.scalameta:semanticdb-scalac_2.13.4:4.3.20`

Reply:
It helped?

Reply:
Ah see. You need to update:
```
"ch.epfl.scala" % "sbt-scalafix" % "0.9.19",
```
in `plugins.sbt`.

Reply:
ok, tnx